wordpress-mobile/WordPress-iOS
# Present NoticeViews in a UIWindow

This is primarily needed to allow us to show `NoticeView`s from the editor. However it also makes some of the notices from Quick Start look better.

![new notice window](https://user-images.githubusercontent.com/517257/49765117-4fd55b80-fc86-11e8-850f-99d0f82d11a9.gif)

To test:
- Ensure that Quick Start notices appear correctly
- Ditto for the notice after publishing a post
- Check on both iPhone and iPad
- Ensure that rotation behaves well

@stevebaranski @jtreanor can you check this out, since both of you worked on `UIWindow` implementations before? Also, @frosty here's a ping so that you aren't caught entirely unaware of this 😉 

Doesn't require a change in the release notes.
